### PR TITLE
[power] - Adds Frequency and Current as an emulated device

### PR DESCRIPTION
**Pull request by lazypower**
- Adds emulation for frequency measurements in HZ to the emulator
  plugin.
- Adds emulation for current measurements in Ampere to the emulator plugin.
- bumps to 2.5.1

### CODE REVIEW

**Comment by lazypower** [comment]
@edaniszewski i tried to do this one solo. can you give this a once over to ensure the nomenclature is correct in the places of instantiation? i didnt know if we wanted to dcall the measurement frequency or hertz. but i opted for the "device name" so its a little more apparent.

File: config/device/power.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ devices:
         location: r1vec
         data:
           id: 1
-  - name: kilawatthour
+  - name: killawatthour
```
Comment (KylerBurke) [inline config/device/power.yaml]: `kilowatt hour`*

Comment (marcoceppi) [inline config/device/power.yaml]: kill**o**watthour?

File: pkg/devices/power.go [hunk 2]
```diff
@@ -8,6 +8,13 @@ import (
...
 )
 
+// Frequency is the handler for the emulated frequency device(s).
+var Frequency = sdk.DeviceHandler{
+	Name:  "frequency",
+	Read:  frequencyRead,
+	Write: frequencyWrite,
+}
+
 // KillaWattHours is the handler for the emulated kwh device(s).
```
Comment (KylerBurke) [inline pkg/devices/power.go]: KiloWattHours

File: pkg/outputs/outputs.go [hunk 3]
```diff
@@ -96,4 +96,14 @@ var (
 			Symbol: "kWh",
 		},
 	}
+
+	// Hertz is the output type for voltage devices in kwh.power.
```
Comment (edaniszewski) [inline pkg/outputs/outputs.go]: voltage -> frequency
kwh.power -> hertz

File: pkg/devices/power.go [hunk 4]
```diff
@@ -8,6 +8,13 @@ import (
...
 var KillaWattHours = sdk.DeviceHandler{
 	Name:  "killawatthour",
```
Comment (KylerBurke) [inline pkg/devices/power.go]: killa killa wat? killa wat wat?? THOUR!

File: pkg/devices/power.go [hunk 5]
```diff
@@ -29,8 +36,88 @@ var Voltage = sdk.DeviceHandler{
 	Write: voltageWrite,
 }
 
+// frequencyRead is the read handler for the emulated frequency device(s).
+// It returns random values between 0 and 4500.
+func frequencyRead(device *sdk.Device) ([]*sdk.Reading, error) {
+	// Default reading ranges
+	var min = 0
+	var max = 60
```
Comment (KylerBurke) [inline pkg/devices/power.go]: max value inconsistent with docstring

File: pkg/outputs/outputs.go (hunk 3 above)
Comment (lazypower) [reply pkg/outputs/outputs.go]: ezgif-5-591305f658de

File: config/device/power.yaml (hunk 1 above)
Comment (KylerBurke) [reply config/device/power.yaml]: with one `l`

**Comment by lazypower** [comment]
I see my schenanigans were not wanted. Which is fine ;) Cleaned up and ready for more scrutiny

**Pull request merged**